src/Aquifer.API/Endpoints/Resources/Content/Publish/Endpoint.cs

```diff
@@ -86,10 +94,35 @@ await Helpers.CreateNewDraft(dbContext,
 
                 await historyService.AddStatusHistoryAsync(mostRecentContentVersion, ResourceContentStatus.Complete, user.Id, ct);
             }
+            
+            var machineTranslation = await dbContext
+                .ResourceContentVersionMachineTranslations
+                .AsTracking()
+                .FirstOrDefaultAsync(x => x.Id == mostRecentContentVersion.Id, ct);
+
+            if (machineTranslation is not null)
+            {
+                similarityScoreMessages.Add(
+                    new ScoreResourceContentVersionSimilarityMessage(
+                        machineTranslation.Id,
+                        mostRecentContentVersion.Id,
+                        ResourceContentVersionSimilarityComparisonTypes.MachineTranslationToResourceContentVersion)
+                );
+            }
+            else
+            {
+                logger.LogInformation($"No machine translation found for published content version {mostRecentContentVersion.Id}.");
```

Comment:
Open to suggestions on how to best log this as we do not want to interrupt the request but we also might be interested in which published resources do not have machine translations.

Reply:
This seems reasonable though I'd add something like `"Skipping similarity scoring."` at the end.